packages/examples/.storybook/preview.js

```diff
@@ -4,6 +4,7 @@ addParameters({
   viewMode: 'story',
   docs: { page: null },
   options: {
+    showPanel: true,
```

Comment:
```suggestion
```

The default value is `true`. I haven't aware the option is not mentioned in Storybook docs since now 😮